libkbfs/journal_server.go

```diff
@@ -551,9 +554,16 @@ func (j *JournalServer) Status(
 	ctx context.Context) (JournalServerStatus, []tlf.ID) {
 	j.lock.RLock()
 	defer j.lock.RUnlock()
-	var totalUnflushedBytes int64
+	var totalStoredBytes, totalUnflushedBytes int64
 	tlfIDs := make([]tlf.ID, 0, len(j.tlfJournals))
 	for _, tlfJournal := range j.tlfJournals {
+		storedBytes, err := tlfJournal.getStoredBytes()
```

Comment:
It would be slightly better to get both the stored and unflushed bytes together under the same journal lock, so they are consistent for each TLF.  So maybe combine `tlfJournal.getStoredBytes()` and `tlfJournal.getUnflushedBytes()` into one function that returns both?